README.md

```diff
@@ -1,152 +1,164 @@
+### 1. Write a Furyfile
 
-- You can get furyctl version with `furyctl version`:
+A `Furyfile` is a simple YAML formatted file that lists which modules (and versions) of the KFD you want to download.
+An example `Furyfile` is the following:
 
-```bash
-$ furyctl version
-INFO[0000] Furyctl version 0.2.3
+```yaml
+# Here you can specify which versions of the modules to use
+versions:
+  networking: v1.7.0
+  monitoring: v1.13.0
+  logging: v1.9.1
+  ingress: v1.11.2
+  dr: v1.8.0
+  opa: v1.5.0
+
+# The bases are a sets of Kustomize bases to deploy Kubernetes components
+bases:
+  - name: networking/
+  - name: monitoring/
+  - name: logging/
+  - name: ingress/
+  - name: dr/
+  - name: opa/
 ```
 
-#### Autocomplete
+In reality, each module it's composed by a set of packages. In the previous `Furyfile`, we downloaded every module's packages. You can cherry-pick single packages using the `module/package` syntax.
 
-You can enable autocompletion for `furyctl` cli on your shell similar to the one
-for `kubectl`. Currently autocompletion is supported for `bash`, `zsh`, `fish`
-and `powershell`. To see the options and info about how to setup, run the
-command:
+A more complete `Furyfile` would be:
 
-``` bash
-$ furyctl completion -h
+```yaml
+# Here you can specify which versions of the modules to use
+versions:
+  networking: v1.7.0
+  monitoring: v1.13.0
+  logging: v1.9.1
+  ingress: v1.11.2
+  dr: v1.8.0
+  opa: v1.5.0
+
+# The bases are a sets of Kustomize bases to deploy Kubernetes components
+bases:
+  - name: networking/calico
+  - name: monitoring/prometheus-operator
+  - name: monitoring/prometheus-operated
+  - name: monitoring/grafana
+  - name: monitoring/goldpinger
+  - name: monitoring/configs
+  - name: monitoring/kubeadm-sm
+  - name: monitoring/kube-proxy-metrics
+  - name: monitoring/kube-state-metrics
+  - name: monitoring/node-exporter
+  - name: monitoring/metrics-server
+  - name: monitoring/alertmanager-operated
+  - name: logging/elasticsearch-single
+  - name: logging/cerebro
+  - name: logging/curator
+  - name: logging/fluentd
+  - name: logging/kibana
+  - name: ingress/cert-manager
+  - name: ingress/nginx
+  - name: ingress/forecastle
+  - name: dr/velero
+  - name: opa/gatekeeper
 ```
 
+You can find out what packages are inside each module by referring to each module documentation.
+
+### 2. Download the modules
 
-### Self-Provisioning
+To download packages, run `furyctl vendor` (within the same directory where your `Furyfile` is located)
+
+`furyctl` will download all the packages in a `vendor/` directory.
+
+## Self-Provisioning
 
 The self-provisioning feature is available with two commands:
 
-- `furyctl bootstrap`: Use it to create the required infrastructure to place the cluster. Skip it if you
+- `furyctl bootstrap`: use it to create the required infrastructure to place the cluster. Skip it if you
 already managed to have passed all the cluster requirements.
 - `furyctl cluster`: Deploys a Fury cluster.
```

Comment:
I wouldn't say that I deploy a cluster, I create a cluster and then deploy stuff on it